src/components/Home.js

```diff
@@ -0,0 +1,76 @@
+import { useState, useEffect } from 'react';
+import { useSelector } from 'react-redux';
+import CountUp from 'react-countup';
+import { FaSearch } from 'react-icons/fa';
+import { IoGlobeSharp } from 'react-icons/io5';
+import ContinentList from './ContinentList';
+
+const Home = () => {
```

Comment:
- [ ] Please change the alignment of the text to the right of each element on the home page to match the design guidelines.

| Your design | Behance design |
| --- | --- |
| ![image](https://user-images.githubusercontent.com/24703171/176435489-185073eb-fc40-4e2f-a316-34113af8e73f.png) | ![image](https://user-images.githubusercontent.com/24703171/176435586-d2aa85df-261b-49ef-9421-ee89c9f70ef0.png) |